.github/workflows/playwright.yml

```diff
@@ -25,3 +26,10 @@ jobs:
         run: npx playwright install --with-deps
       - name: Run ${{ matrix.test-type }} tests on ${{ matrix.project }}
         run: npm run playwright-${{ matrix.test-type }} -- --project ${{ matrix.project }}
+      - name: Upload playwright report
+        if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: playwright-report-${{ matrix.project }}-${{ matrix.test-type }}
+          path: playwright-report/
```

Comment:
This seems wrong, given the new report configs.

Reply:
It works, see https://github.com/zakodium-oss/react-science/actions/runs/7399720269